frontend/src/locales/scripts/en.json5

```diff
@@ -467,10 +467,24 @@
       },
     },
   },
-  generatedTags: {
-    title: "About Generated Tags",
+  tags: {
+    title: "Understanding Tags in {openverse}",
+    /** generatedTags.intro.a-b are parts of a single section explaining how tags work in Openverse. */
     intro: {
-      content: "Generated tags are automatically assigned to media items based on the content of the media item itself. These tags are generated using machine learning models trained on a large dataset of images and audio tracks.",
+      a: "Each creative work in {openverse} includes tags, an optional set of keywords used to describe the work and make it easier for users to find what they are looking for.",
```

Comment:
Maybe this should be "may have tags" instead of "include tags", since the latter makes it seem like each record will have tags (even though it says optional right after that).